android/build-envoy.sh

```diff
@@ -38,6 +38,7 @@ set -o pipefail
 
 BUILD=${1:-release}
 cp "../native/cronet-$BUILD.aar" ./cronet/
+cp "../native/cronet-$BUILD.aar" ./demo/cronet/
```

Comment:
i think this also needs to go in ./envoy/cronet/ instead of just ./cronet  does it still build correctly if you clear out all the existing .aars first?

Reply:
I think I had that change and lost it... I'll verify and correct if needed